components/loading-box/src/index.js

```diff
@@ -0,0 +1,166 @@
+  bottom: 0,
+  display: 'flex',
+  justifyContent: 'center',
+  ' > svg': {
+    position: 'absolute',
+    zIndex: 1,
+    opacity: 1,
+    display: 'block',
+    height: '100%',
+    maxHeight: 'calc(50vh + 100px)',
+    transition: `opacity ${timeIn}ms ease-in-out`,
+  },
+  ' > div': {
+    transition: `background-color ${timeIn}ms ease-in-out`,
+    backgroundColor: `rgba(
+      ${hexRgb(backgroundColor).red},
+      ${hexRgb(backgroundColor).green},
+      ${hexRgb(backgroundColor).blue},
+      ${backgroundColorOpacity})`,
+  },
+  '.fade-enter': {
+    ' > div': {
+      backgroundColor: `rgba(
+        ${hexRgb(backgroundColor).red},
+        ${hexRgb(backgroundColor).green},
+        ${hexRgb(backgroundColor).blue},
+        0)`,
+      transitionDuration: `${timeIn}ms`,
+    },
+    ' svg': {
+      opacity: 0,
+      transitionDuration: `${timeIn}ms`,
+      transitionDelay: `${timeIn / 2}ms`,
+    },
+  },
+  '.fade-enter-active': {
+    ' > div': {
+      backgroundColor: `rgba(
+        ${hexRgb(backgroundColor).red},
+        ${hexRgb(backgroundColor).green},
+        ${hexRgb(backgroundColor).blue},
+        ${backgroundColorOpacity})`,
+      transitionDuration: `${timeIn}ms`,
+    },
+    ' svg': {
+      opacity: 1,
+      transitionDuration: `${timeIn}ms`,
+      transitionDelay: `${timeIn / 2}ms`,
+    },
+  },
+  '.fade-exit': {
+    ' > div': {
+      backgroundColor: `rgba(
+        ${hexRgb(backgroundColor).red},
+        ${hexRgb(backgroundColor).green},
+        ${hexRgb(backgroundColor).blue},
+        ${backgroundColorOpacity})`,
+      transitionDuration: `${timeOut}ms`,
+    },
+    ' svg': {
+      opacity: 1,
+      transitionDuration: `${timeOut}ms`,
+    },
+  },
+  '.fade-exit-active': {
+    ' > div': {
+      backgroundColor: `rgba(
+        ${hexRgb(backgroundColor).red},
+        ${hexRgb(backgroundColor).green},
+        ${hexRgb(backgroundColor).blue},
+        0)`,
+      transitionDuration: `${timeOut}ms`,
+    },
+    ' svg': {
+      opacity: 0,
+      transitionDuration: `${timeOut}ms`,
+    },
+  },
+}));
+
+const Overlay = glamorous.div({
+  position: 'absolute',
+  top: 0,
+  right: 0,
+  bottom: 0,
+  left: 0,
+  overflow: 'hidden',
+  height: '100%',
+  width: '100%',
+});
+
+const LoadingBox = ({
+  children,
+  backgroundColor,
+  backgroundColorOpacity,
+  loading,
+  spinnerColor,
+  timeIn,
+  timeOut,
+  ...props
+}) => (
+  <Wrapper>
+    <CSSTransition {...props} timeout={timeOut} classNames="fade" in={loading} unmountOnExit>
+      <Innerwrap
+        backgroundColor={backgroundColor}
+        backgroundColorOpacity={backgroundColorOpacity}
+        loading={loading}
+        timeIn={timeIn}
+        timeOut={timeOut}
+      >
+        <Spinner fill={spinnerColor} width="100px" height="100px" />
+        <Overlay />
+      </Innerwrap>
+    </CSSTransition>
+    {children}
+  </Wrapper>
+);
+
+LoadingBox.defaultProps = {
+  spinnerColor: WHITE,
+  backgroundColor: '#808080',
```

Comment:
50% gray ✅

Reply:
* grey

Reply:
did we discuss making this white by default?

Reply:
yeah but you said this was configurable in with knobs anyway

Reply:
i will change to white